### DISCUSSION TITLE
Rabbitmq connection

### DISCUSSION THREAD

**Discussion by hasifasi** [learndevopsonline/learndevopsonline#1982] [Q&A]
Hi

This is my rabbitmq serrver: 34.236.152.225
this is my dispatch server : 54.173.156.236 

rabbitmq is active and also given its info in dispatch.service...

unfortunatly , dispatch unable to connect to rabbitmq.

frontend server: 34.205.76.12

**Comment by hasifasi** [comment]
This is the log error

Oct  5 18:54:35 ip-172-31-18-244 dispatch[5435]: 2024/10/05 18:54:35 dial tcp: lookup RABBITMQ-IP on 172.31.0.2:53: no such host
Oct  5 18:54:35 ip-172-31-18-244 dispatch[5435]: 2024/10/05 18:54:35 Reconnecting to amqp://roboshop:roboshop123@RABBITMQ-IP:5672/
Oct  5 18:54:36 ip-172-31-18-244 dispatch[5435]: 2024/10/05 18:54:36 dial tcp: lookup RABBITMQ-IP on 172.31.0.2:53: no such host
Oct  5 18:54:36 ip-172-31-18-244 dispatch[5435]: 2024/10/05 18:54:36 Reconnecting to amqp://roboshop:roboshop123@RABBITMQ-IP:5672/

**Comment by hasifasi** [comment]
Hi Any help on this?

**Reply by chaithanya1812** [reply]
check once I just restarted the dispatch, it's working fine now..

**Reply by hasifasi** [reply]
you mean you did ' systemctl restart dispatch'... if so, weird, i tried that option many times and didnt help at all...

**Reply by hasifasi** [reply]
now it works when u restart it.. but weird though
